setup.py

```diff
@@ -46,7 +46,7 @@
     include_package_data=True,
     python_requires=">=3.6",
     install_requires=[
-        "Django>=3.0,<3.2",
+        "Django>=3.0",
```

Comment:
I would give it a few passes with only local Django version changes before going this far with it, especially while we have older dependencies like PyXB.

Reply:
I updated it to be between 3.2 and 4.0.